views/js/controller/Delivery/monitoring.js

```diff
@@ -516,6 +518,96 @@ define([
                 }
 
                 /**
+                 * Formats a timer
+                 * @param {Number} totalSeconds
+                 * @returns {String}
+                 */
+                function formatTime(totalSeconds) {
+                    var sec_num = totalSeconds;
+                    var hours = Math.floor(sec_num / 3600);
+                    var minutes = Math.floor((sec_num - (hours * 3600)) / 60);
+                    var seconds = Math.floor(sec_num - (hours * 3600) - (minutes * 60));
+
+                    if (hours < 10) {
+                        hours = "0" + hours;
+                    }
+                    if (minutes < 10) {
+                        minutes = "0" + minutes;
+                    }
+                    if (seconds < 10) {
+                        seconds = "0" + seconds;
+                    }
+
+                    return hours + ':' + minutes + ':' + seconds;
+                }
+
+                function updateTimer(rows) {
```

Comment:
You shouldn't code a new timer countdown but reuse the one available.
See `core/polling` and `core/timer`. The  [timer plugin](https://github.com/oat-sa/extension-tao-testqti/blob/master/views/js/runner/plugins/controls/timer/timer.js#L404) has a good example.